src/game.c

```diff
@@ -204,3 +206,41 @@ void handle_computer_turn(void)
     return;
 }
 
+/**
+ * @brief Draw a new card and add it to another player's cards on hand list.
```

Comment:
Can you change this to  "add it to the next player" ?

Reply:
Done.